src/main/java/com/pinterest/secor/uploader/S3UploadManager.java

```diff
@@ -81,6 +82,7 @@ public S3UploadManager(SecorConfig config) {
 
         final String accessKey = mConfig.getAwsAccessKey();
         final String secretKey = mConfig.getAwsSecretKey();
+        final String sessionToken = mConfig.getAwsSessionToken();
```

Comment:
If the config is like 'aws.session.token=', will line 85 throw exception?

Reply:
It runs fine for me with it being set empty like that. Will just be an empty string similar to access key, secret key and role when they are not set